tests/aws/services/kms/test_kms.snapshot.json

```diff
@@ -1579,7 +1579,7 @@
       "decrypt_response_with_encryption_context": {
         "EncryptionAlgorithm": "SYMMETRIC_DEFAULT",
         "KeyId": "<key-id:1>",
-        "KeyMaterialId": "e2333676b9bf055cb0caa2bec3957d7f3e60b7545a3706314e397746cd26122e",
+        "KeyMaterialId": "43d71366b52c0dddc21719cdd087369c1fd3cdd3a7075e33e1db6f5e60d6d30e",
```

Comment:
nit: will transform the value, already have the commit but waiting for the full run before pushing